### PR TITLE
fix: generate-certificates namespace

### PR DESCRIPTION
**Pull request by JorickPepin**

### CODE REVIEW

File: .castor/docker.php [hunk 1]
```diff
@@ -150,7 +150,7 @@ function destroy(
     fs()->remove($files);
 }
 
-#[AsTask(description: 'Generates SSL certificates (with mkcert if available or self-signed if not)', namespace: '')]
+#[AsTask(description: 'Generates SSL certificates (with mkcert if available or self-signed if not)', aliases: ['generate-certificates'])]
```
Comment (lyrixx) [inline .castor/docker.php:153]: Hmm, I don't think it's really useful. We run this comment only once (and actually never because the very first time on a projet, you run `castor start`)

It might conflict with other commands.

Could you revert this changes?

**Review by lyrixx — COMMENTED** [review]
Good catch! thanks. I let one little comment tho

**Review by pyrech — APPROVED** [review]
Thanks
